tests/e2e/go.mod

```diff
@@ -0,0 +1,47 @@
+module k8s.io/cloud-provider-aws/tests/e2e
+
+go 1.16
+
+require (
+	github.com/google/uuid v1.1.4 // indirect
+	github.com/imdario/mergo v0.3.11 // indirect
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.3
+	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
+	k8s.io/api v0.22.0
+	k8s.io/apimachinery v0.22.0
+	k8s.io/client-go v9.0.0+incompatible
+	k8s.io/kubectl v0.17.2 // indirect
+	k8s.io/kubernetes v1.22.0
```

Comment:
i find/replaced 23.0 with 22.0. probably could use a patch release, don't think it much matters

Reply:
perhaps not since its just test framework dependencies